composer.json

```diff
@@ -12,6 +12,9 @@
     "magento/module-quote": ">=101.1.1 <101.2.8",
     "magento/module-store": ">=101.0.1 <101.1.8"
   },
+  "require-dev": {
+    "joseph-leedy/module-custom-fees": "*"
```

Comment:
Not sure how I feel about including this as a dev requirement. I know we needed it to be able to test that fees were working as expected but including it here feels like too much. Are you using this in your integration test?

Reply:
Yes, that was why I included it.

Reply:
My concern over this is partially related to the fees module not being under Bold's github space. Is the plan to have this custom fees module live under Bold's umbrella or will it only exist in your personal github? 

I think it's valid to want to ensure that this feature works and is tested properly but I'm hesitant to rely on a package that isn't owned by Bold for tests to pass. Even though this is a dev dependency I'm not sure if security vetting would be a concern on top of this as well

Reply:
I put it in my personal account because I plan to open source the extension (in fact, I've already created a [repository](https://github.com/JosephLeedy/magento2-module-custom-fees) with a skeleton extension* and put it on Packagist).

Regarding security, that should not be an issue because a lot of the code was "borrowed" from the core WEEE extension, and it doesn't touch anything that would be in PCI scope. It's also not a package that would be installed by normal end-users of the final product (Payment Booster).

* I plan to clean up the code and commit it in my own time over the next few weeks.